src/code/qualification/plotQualificationGOFMerged.m

```diff
@@ -139,8 +139,9 @@
     end
     legendLabels=[legendLabels Groups(i).Caption];
 end
-xLabelFinal = getLabelWithUnit('Observations',xAxesOptions.Unit);
-yLabelFinal = getLabelWithUnit('Predictions',yAxesOptions.Unit);
+
+xLabelFinal = getLabelWithUnit(sprintf('Observed %s', findDimensionfromUnit(xAxesOptions.Unit)),xAxesOptions.Unit);
+yLabelFinal = getLabelWithUnit(sprintf('Simulated %s', findDimensionfromUnit(yAxesOptions.Unit)),yAxesOptions.Unit);

```

Comment:
Missing 2nd format argument in both sprintfs: `sprintf('Observed %s %s ', ...`

Reply:
Since I use the function getLabelWithUnit, the second argument which should be the unit is not in sprintf.
This way it will be in [ ] if there is a unit and there won't be [ ] if there is no unit.

Reply:
Ok, I see.
Everything good then, PR can be merged as soon as #216 was merged.